packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1300,7 +1300,7 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
     assert(child != null);
     assert(child.parentData != null);
     child._cleanRelayoutBoundary();
-    child.parentData.detach();
+    child.parentData!.detach();
     child.parentData = null;
     super.dropChild(child);
     markNeedsLayout();
```

Comment:
let's change debugCreator below to `Object?`

Reply:
done